clang/test/CodeGenSYCL/regcall-cc-test.cpp

```diff
@@ -0,0 +1,106 @@
+// RUN:  %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -disable-llvm-passes -O0 -emit-llvm %s -o - | FileCheck %s
```

Comment:
After #3929, we don't need sycldevice in the target triple.

Reply:
```suggestion
// RUN:  %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -disable-llvm-passes -O0 -emit-llvm %s -o - | FileCheck %s
```